examples/dashboard.ipynb

```diff
@@ -0,0 +1,139 @@
+    "import intake, geoviews.tile_sources as gts\n",
+    "from holoviews.operation.datashader import rasterize, shade, spread\n",
+    "\n",
+    "hv.extension('bokeh', logo=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can run the dashboard here in the notebook with various datasets by editing the `dataset` below to specify some dataset defined in `dashboard.yml`.  You can also launch a separate, standalone server process in a new browser tab with a command like:\n",
+    "\n",
+    "```\n",
+    "DS_DATASET=nyc_taxi panel serve --show dashboard.ipynb\n",
+    "DS_DATASET=census   panel serve --show dashboard.ipynb\n",
+    "DS_DATASET=opensky  panel serve --show dashboard.ipynb\n",
+    "DS_DATASET=osm      panel serve --show dashboard.ipynb\n",
+    "```\n",
+    "\n",
+    "To launch multiple dashboards at once, you'll need to add `-p 5001` (etc.) to select a unique port number for the web page to use for communicating with the Bokeh server.  Otherwise, be sure to kill the server process before launching another instance.\n",
+    "\n",
+    "For most of these datasets, if you have less than 16GB of RAM on your machine, you should remove the `.persist()` call below, to tell [Dask](http://dask.pydata.org) to work out of core instead of loading all data into memory.  However, doing so will make interactive use substantially slower than if sufficient memory were available."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# nyc_taxi, census, opensky, osm-1billion\n",
+    "dataset = os.getenv(\"DS_DATASET\", \"nyc_taxi\")\n",
+    "catalog = intake.open_catalog('dashboard.yml')\n",
+    "source  = getattr(catalog, dataset)\n",
+    "source.to_dask().persist();"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Intake `source` object lets us treat data in many different formats the same in the rest of the code here. We can now build a class that captures some parameters that the user can vary along with how those parameters relate to the code needed to update the displayed plot of that data source:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gopts = hv.opts.WMTS(width=800, height=500, xaxis=None, yaxis=None, bgcolor='black', show_grid=False)\n",
+    "cmaps = {n: colorcet.palette[n] for n in ['fire', 'bgy', 'bgyw', 'bmy', 'gray', 'kbc']}\n",
+    "norms = {'Histogram_Equalization': 'eq_hist', 'Linear': 'linear', 'Log': 'log', 'Cube root': 'cbrt'}\n",
+    "maps  = ['CartoMidnight', 'StamenWatercolor', 'StamenTonerBackground', 'EsriImagery', 'EsriUSATopo', 'EsriTerrain']\n",
+    "bases = {name: ts.relabel(name) for name, ts in gts.tile_sources.items() if name in maps}\n",
+    "plots = {source.metadata['plots'][p].get('label', p):p for p in source.plots}\n",
```

Comment:
So just limit the list of fields to ones that you want to allow aggregation over and have one just be counts or something if you want to allow that. 

This will get a dict of labelled selectors for all the available aggregator fields. 

```suggestion
    "plots = {source.metadata['plots'][p].get('label', p):p for p in source.plots}\n",
    "fields = {k: v.get('label', k), for k, v in source.metadata['fields'].items()}\n",
```